### DISCUSSION TITLE
Battle #13 - Used technics discussion

### DISCUSSION THREAD

**Discussion by Beowolve** [cssbattle/forum#39] [General]
I would like to start some discussion about used technics of the none public targets.

- Froggy #70:
I made two good versions one with unicode and one without, but both ended up with exactly the same character count.
Both use radial-gradient for the main part, and one version even uses webkit-box-reflect. I tried to get rid of it, but without success.

- Sheep #72: 
This one is quite unique for me, because it uses only text-shadow and matrix transformations with a lot of unicode characters

- Danger Noodle #74:
This one manly uses liner- and radial-gradients in my solution. I basically made all vertical lines with one background gradient and overlapped the needed parts with the radial gradients. No unicode used here.

- Hippo #75:
box-shadow and some matrix transformations as well. Did anyone manage to use just one border-radius for all elements? I always ended up with 99.9%, but it looks possible somehow. 64/51 is so close...

- Beeee #76:
Most characters are consumed by the linear-gradient for the body and I used a unicode character for the eye.
Is there a shorter version of this?
`repeating-linear-gradient(90deg,#191919 0 26q,0,#0000 37q)no-repeat 0/1in#EFF33C`

**Comment by chrona** [comment]
* Froggy #70
  Used radial-gradient for the face and box-shadow for the eyes+nose
  
* Sheep #72
  This one… First draft was basically two elements - one of them rotated - to make the background and just add the face on top. After that i tried to use `❋` with a lot of text-stroke, transform and font-sizes but couldn't get it to 100%
  
* Danger Noodle #74
  First attempt used box-shadow with clip-path, but rebuild it using border instead of box-shadow
  
* Hippo #75
  Basically border-radius and transformations as well
  
* Beeee #76
  Didn't think of using unicode for the eye 😐 Ended up with four elements (2 wings + 2 body parts) and changed border-radius/background for them & one extra element for the eye

**Comment by jbarrantes** [comment]
- Danger Noodle #74:
Same approach, the linear gradient and the radial gradient both share the same pattern in my case via var(--)

- Beeee #76:
For body I used 2 gradients but couldn't further reduce the char count better than your solution.
By the way, did you used 2 shadows for the wings?

**Reply by Beowolve** [reply]
>   By the way, did you used 2 shadows for the wings?

Shadows? I am confused, how would I use that to create that shape. I used the classic method with border-radius:50%50%0 for one wing and the inverse for the other wing. Using box-reflect for the second wing ended up using one extra character for me.

**Comment by Vuurvos1** [comment]
- Froggy #70:
  The same solution (gradient and box-shadow)

- Sheep #72:
  For this one, I created the "wool" circle first and then put the face behind it. A combination of transform, border-radius and border

- Danger Noodle #74:
  Box shadow and clip-path in combination with some transforms

- Hippo #75:
  A relatively basic solution using transform, border and border-radius

- Beeee #76:
  Gradient for the body and border-radius for the rounded parts

**Comment by SelenIT** [comment]
The best currently working non-Unicode non-SVG solutions I could come up with:

[#69 PushOwl](https://cssbattle.dev/play/69)
```
<center><a><a><style>* *{margin:125-3;--g:#191919 75%,#E08027 0;background:conic-gradient(from.8rad at 50%75q,var(--g))0 4cm}a+a{--a:50%0;box-shadow:0 0 0 10PX#191919}a{padding:48.5 57;border-radius:var(--a,0)50%50%;background:radial-gradient(9q at bottom,var(--g)16q,#0000)0-54Q,radial-gradient(64q,var(--g
```

[#70 Froggy](https://cssbattle.dev/play/70)
```
<img><img><a><a><style>a{padding:2 5;margin:0 29 0-59}img{padding:25;margin:-25 4 32 16}body{font:1%a;--s:-8pc}*{margin:55 62 45;--s:;border-radius:1in;background:radial-gradient(1q at 50%var(--s),#293462 0 32em,#FFF1C1 0 5vh,#FE5F55 0 210q,#A64942
```

[#72 Sheep](https://cssbattle.dev/play/72) (I was kinda proud of this one 😳)
```
<body bgcolor=243d83 text=6592d><dl><dt><dl><a><img><style>*+*{margin:125 170}dl{--s:64q,0-64q}a{padding:33 7 22;--s:-20q 0 20q}img{padding:5;margin:26 18 10;--s:15q,15q 0}* *{--s:60q,0-60q;border-radius:65%;rotate:45deg;box-shadow:0 var(--s
```

[#74 Danger Noodle](https://cssbattle.dev/play/74) (look ma, just one gradient for anything! And two `i` elements from one `<i>` tag)
```
<p><b><i><img></b><p>.<style>p{--r:106q 3in}p+p{margin:59%0;--p:96%153q;padding:56%0}p *{padding:95%0 99 43}b{margin:200}b i{--r:2%3in}img{margin:135-23-96;padding:15 26;border-radius:16q;--r:140%56%;font:33%a}*+*{--r:39%3q;margin:0 36%0 0}*{background:var(--p,0-73q)/var(--r,4in)radial-gradient(2em,#191919 48%,#E08027 0 5vw,#191919 0 95%,#E08027 0 37q,#191919
```

[#75 Hippo](https://cssbattle.dev/play/75) (looks far from ideal, but still accepted 😳)
```
<img><p><img><style>* *{--m:70-10-20;--s:2in#A64942}img{padding:11 5;--m:-70 109;--s:5q#FE5F55,30q 17q 0 3q#FE5F55,30q 30q,11q 129q 0 4px,0 19q 0-9q;rotate:45deg}p{scale:-1 1;--m:}*{box-shadow:0 0 0 var(--s,4in#191919,0 2in#FE5F55)inset;border-radius:63q 63q 52q 52q;margin:var(--m,75 135
```

[#76 Beeee](https://cssbattle.dev/play/76)
```
<a><a><p><a><a><a i><style>a{--p:75 37.5 0}p{--b:#EFF33C;--p:27 0 48}p a{--p:80 12.5;--b:#191919;margin:0 10 0 0}[i]{--p:7;margin:11}* *{padding:var(--p,142 117 0);font:0 a;clip-path:inset(0 round 41q);background:var(--b,radial-gradient(1q,#fff,41q,#998235
```

**Reply by Beowolve** [reply]
Looks like #69 isn't working with .8rad anymore :(
